src/Compilers/CSharp/Portable/Binder/RefSafetyAnalysis.cs

```diff
@@ -696,62 +720,126 @@ private void SetPatternLocalScopes(BoundObjectPattern pattern)
             return null;
         }
 
-        private void VisitArgumentsAndGetArgumentPlaceholders(BoundExpression? receiverOpt, ImmutableArray<BoundExpression> arguments)
+        private void VisitArgumentsAndGetArgumentPlaceholders(BoundExpression? receiverOpt, ImmutableArray<BoundExpression> arguments, bool isNewExtensionMethod)
         {
             for (int i = 0; i < arguments.Length; i++)
             {
                 var arg = arguments[i];
                 if (arg is BoundConversion { ConversionKind: ConversionKind.InterpolatedStringHandler, Operand: BoundInterpolatedString or BoundBinaryOperator } conversion)
                 {
                     var interpolationData = conversion.Operand.GetInterpolatedStringHandlerData();
-                    var placeholders = ArrayBuilder<(BoundValuePlaceholderBase, SafeContext)>.GetInstance();
-                    GetInterpolatedStringPlaceholders(placeholders, interpolationData, receiverOpt, i, arguments);
+                    var placeholders = ArrayBuilder<(BoundValuePlaceholderBase, SafeContextAndLocation)>.GetInstance();
+                    GetInterpolatedStringPlaceholders(placeholders, interpolationData, receiverOpt, i, arguments, isNewExtensionMethod);
                     _ = new PlaceholderRegion(this, placeholders);
                 }
                 Visit(arg);
             }
         }
 
+        public sealed override BoundNode? VisitCall(BoundCall node)
```

Comment:
Is this PR fixing some observable issue we should have a test for?

Reply:
No. This override was previously not needed because invocations were rewritten later in the pipeline. Now they're rewritten here.